Soujanya2004/wanderlust-2024
# Improve UI for Review showing section with responsiveness (#16)

Fixes #16

This PR improves the UI for the review showing section with responsiveness.

### Changes
- Added hover effects to the delete button.
- Enhanced spacing, padding, and font styles for better readability and aesthetics.

### Testing Instructions
1. Pull this branch.
2. Run npm install to install dependencies.
3. Run the project and navigate to the review section.
4. Hover over the delete button and review cards to verify the animations and hover effects.
5. Verify the updated UI, padding, and spacing in the review section.

### Screenshots (if applicable)

https://github.com/user-attachments/assets/162c7dc0-b7ed-4d67-beb5-8696f3cc3c15



### Checklist
- [ ]  The PR follows the contribution guidelines.
